modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc

```diff
@@ -0,0 +1,223 @@
+
+. Generate a base64 string of the `~/control-plane-chrony.conf` file.
++
+[source,bash]
+----
+$ base64 ~/control-plane-chrony.conf
+----
++
+The output will look something like this:
++
+[source,bash]
+----
+IyBVc2UgcHVibGljIHNlcnZlcnMgZnJvbSB0aGUgcG9vbC5udHAub3JnIHByb2plY3QuCiMgUGxl
+YXNlIGNvbnNpZGVyIGpvaW5pbmcgdGhlIHBvb2wgKGh0dHBzOi8vd3d3LnBvb2wubnRwLm9yZy9q
+b2luLmh0bWwpLgoKIyBUaGlzIGZpbGUgaXMgbWFuYWdlZCBieSB0aGUgbWFjaGluZSBjb25maWcg
+b3BlcmF0b3IKc2VydmVyIG9wZW5zaGlmdC1tYXN0ZXItMC48Y2x1c3Rlci1uYW1lPi48ZG9tYWlu
+PiBpYnVyc3QKc2VydmVyIG9wZW5zaGlmdC1tYXN0ZXItMS48Y2x1c3Rlci1uYW1lPi48ZG9tYWlu
+PiBpYnVyc3QKc2VydmVyIG9wZW5zaGlmdC1tYXN0ZXItMi48Y2x1c3Rlci1uYW1lPi48ZG9tYWlu
+PiBpYnVyc3QKCnN0cmF0dW13ZWlnaHQgMApkcmlmdGZpbGUgL3Zhci9saWIvY2hyb255L2RyaWZ0
+CnJ0Y3N5bmMKbWFrZXN0ZXAgMTAgMwpiaW5kY21kYWRkcmVzcyAxMjcuMC4wLjEKYmluZGNtZGFk
+ZHJlc3MgOjoxCmtleWZpbGUgL2V0Yy9jaHJvbnkua2V5cwpjb21tYW5ka2V5IDEKZ2VuZXJhdGVj
+b21tYW5ka2V5Cm5vY2xpZW50bG9nCmxvZ2NoYW5nZSAwLjUKbG9nZGlyIC92YXIvbG9nL2Nocm9u
+eQoKIyBDb25maWd1cmUgdGhlIGNvbnRyb2wgcGxhbmUgbm9kZXMgdG8gc2VydmUgYXMgbG9jYWwg
+TlRQIHNlcnZlcnMKIyBmb3IgYWxsIHdvcmtlciBub2RlcywgZXZlbiBpZiB0aGV5IGFyZSBub3Qg
+aW4gc3luYyB3aXRoIGFuCiMgdXBzdHJlYW0gTlRQIHNlcnZlci4KCiMgQWxsb3cgTlRQIGNsaWVu
+dCBhY2Nlc3MgZnJvbSB0aGUgbG9jYWwgbmV0d29yay4KYWxsb3cgYWxsCiMgU2VydmUgdGltZSBl
+dmVuIGlmIG5vdCBzeW5jaHJvbml6ZWQgdG8gYSB0aW1lIHNvdXJjZS4KbG9jYWwgc3RyYXR1bSAz
+IG9ycGhhbgo=
+----
++
+Replace the `BASE64ENCODEDCONFIGFILE` string in the `~/ntp-server.yaml` with the base64 encoded string.
+
+. Apply the `ntp-server.yaml` policy to the control plane nodes.
++
+[source,bash]
+----
+$ oc apply -f ~/ntp-server.yaml
+----
++
+[source,bash]
+----
+machineconfig.machineconfiguration.openshift.io/99-master-etc-chrony-conf-override-for-server created
+----
+
+. Create a `~/ntp-client.yaml` file for telling the Machine Configuration Operator to apply the `~/worker-chrony.conf` settings to the NTP clients on the worker nodes.
++
+[source,bash]
+----
+# This example MachineConfig replaces ~/worker-chrony.conf
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-master-etc-chrony-conf-override-for-worker
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,BASE64ENCODEDCONFIGFILE<1>
+          filesystem: root
+          mode: 0644
+          path: /etc/worker-chrony.conf
+----
++
+Where:
++
+<1> You must replace the `BASE64ENCODEDCONFIGFILE` string with the base64 encoded string of the `~/worker-chrony.conf` file in the subsequent step.
+
+
+. Generate a base64 encoded string of the `~/worker-chrony.conf` file.
++
+[source,bash]
+----
+$ base64 ~/worker-chrony.conf
+----
++
+The output will looks something like this:
```

Comment:
I would remove "The output will looks something like this:" and just use this before the code block:

    .Example output